hathor/manager.py

```diff
@@ -423,14 +423,15 @@ def validate_new_tx(self, tx: BaseTransaction) -> bool:
             tx = cast(Block, tx)
             assert tx.hash is not None  # XXX: it appears that after casting this assert "casting" is lost
 
-            # Validate minimum block difficulty
-            block_weight = self.calculate_block_difficulty(tx)
-            if tx.weight < block_weight - settings.WEIGHT_TOL:
-                raise InvalidNewTransaction(
-                    'Invalid new block {}: weight ({}) is smaller than the minimum weight ({})'.format(
-                        tx.hash.hex(), tx.weight, block_weight
+            if self.state != self.NodeState.INITIALIZING:
```

Comment:
I'd just add a comment explaining that we can skip this check during startup as we'll trust the difficulty stored in metadata.

Reply:
Done